test/e2e/network/ingress.go

```diff
@@ -101,6 +101,8 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 			}
 			err := gceController.Init()
 			framework.ExpectNoError(err)
+			ipAddress = gceController.CreateStaticIP(ns)
```

Comment:
I think you have to declare this variable just below ` var gceController`

Reply:
ooooh completelly forget and vscode trolled me! Thanks Antonio!

Reply:
ups, but it' s being used only here just use ` :=`

Reply:
Yeah actually I've followed the same way it's been done in further part of the code (there's another beforeeach with allocate) and have declared the variable outside the function :)

But it's up to you where is this better :)